db/migrate/20210806090342_create_movies.rb

```diff
@@ -0,0 +1,11 @@
+class CreateMovies < ActiveRecord::Migration[6.1]
+  def change
+    create_table :movies do |t|
+      t.string :title
+      t.string :description
+      t.string :duration_min
```

Comment:
Seems to be a perfect candidate for a [interval](https://www.bigbinary.com/blog/rails-6-1-adds-postgresql-interval-data-type) type